build.gradle.kts

```diff
@@ -19,228 +19,216 @@ import org.jreleaser.model.Active
 import org.jreleaser.model.Distribution.DistributionType
 import org.jreleaser.model.Stereotype
 import java.util.Properties
-
 import java.util.regex.Pattern
 
-buildscript {
-  repositories {
-    maven { url = uri("https://plugins.gradle.org/m2/") }
-    mavenLocal()
-  }
-}
+group = "software.amazon.smithy"
+version = file("VERSION").readText().trim()
+description = "Language Server Protocol implementation for Smithy"
 
+println("Smithy Language Server version: '${version}'")
 
-plugins {
-    // Apply the java plugin to add support for Java
-    id("java")
+val stagingDirectory = layout.buildDirectory.dir("staging")
+val releaseChangelogFile = layout.buildDirectory.file("resources/RELEASE_CHANGELOG.md").get()
+val smithyVersion: String by project
 
-    // Apply the application plugin to add support for building a CLI application.
-    id("application")
-
-    id("maven-publish")
-    id("com.palantir.git-version") version "0.12.3"
-    id("checkstyle")
-    id("org.jreleaser") version "1.13.0"
+plugins {
+    java
+    application
+    `maven-publish`
+    checkstyle
 
     // Fork of runtime plugin with java 21 support, until https://github.com/beryx/badass-runtime-plugin/issues/153
     // is resolved.
     id("com.dua3.gradle.runtime") version "1.13.1-patch-1"
-}
-
 
-val gitVersion: groovy.lang.Closure<String> by extra
-version = gitVersion().replaceFirst("v", "")
-
-// Reusable license copySpec for building JARs
-val licenseSpec = copySpec {
-    from("${project.rootDir}/LICENSE")
-    from("${project.rootDir}/NOTICE")
-}
-
-// Set up tasks that build source and javadoc jars.
-tasks.register<Jar>("sourcesJar") {
-    metaInf.with(licenseSpec)
-    from(sourceSets.main.get().allJava)
-    archiveClassifier = "sources"
+    id("org.jreleaser") version "1.13.0"
 }
 
-// Build a javadoc JAR too.
-tasks.register<Jar>("javadocJar") {
-    metaInf.with(licenseSpec)
-    from(tasks.javadoc)
-    archiveClassifier = "javadoc"
+repositories {
+    mavenLocal()
+    mavenCentral()
 }
 
-val libraryVersion = project.file("VERSION").readText().trim()
-val imageJreVersion = "21"
-val correttoRoot = "https://corretto.aws/downloads/latest/amazon-corretto-${imageJreVersion}"
-
-println("Smithy Language Server version: '${libraryVersion}'")
-
-val stagingDirectory = rootProject.layout.buildDirectory.dir("staging")
+dependencies {
+    implementation("org.eclipse.lsp4j:org.eclipse.lsp4j:0.23.1")
+    implementation("software.amazon.smithy:smithy-build:[$smithyVersion, 2.0[")
+    implementation("software.amazon.smithy:smithy-cli:[$smithyVersion, 2.0[")
+    implementation("software.amazon.smithy:smithy-model:[$smithyVersion, 2.0[")
+    implementation("software.amazon.smithy:smithy-syntax:[$smithyVersion, 2.0[")
```

Comment:
You don't really need to set the version if you're using a range, just `[1.0, 2.0]`